model/project_ref.go

```diff
@@ -878,6 +883,17 @@ func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) error {
 	return nil
 }
 
+func (p *ProjectRef) AddGitCloneToWorkstationCommands() {
+	if !p.WorkstationConfig.GitClone {
+		return
+	}
+	newCmd := WorkstationSetupCommand{}
+	newCmd.Command = fmt.Sprintf("git clone -b %s https://github.com/%s/%s.git", p.Branch, p.Owner, p.Repo)
```

Comment:
I'm also worried about the token here. We should document it?

Reply:
As in document that this is the method of cloning?

Reply:
yeah, on the UI?